Defs/Ammo/Modded/Warhammer 40k/MeltaGun.xml

```diff
@@ -47,4 +91,58 @@
 		</modExtensions>
 	</ThingDef>
 
+	<!-- ==================== Recipes ========================== -->
+		<RecipeDef ParentName="AmmoRecipeBase">
+		<defName>MakeAmmo_MeltaGun</defName>
+		<label>make meltagun canisters x20</label>
+		<description>Craft 20 melta gun power cells</description>
+		<jobString>Making 20 shots worth of melta gun power cells</jobString>
```

Comment:
Kind of weird wording here, would probably just leave it as "meltagun canisters".